### ISSUE TITLE
getSprite seems to have an invalid return type

### ISSUE THREAD

**Issue by mflerackers** [marklovers/kaplay#173]
The current function signature is
```ts
getSprite(name: string): Asset<SpriteData> | void
```
Which looks invalid, a function either returns something, or not, you can't have both. This causes problems when writing either of these
```ts
getSprite("bean").then()
getSprite("bean")?.then()
```
Since in the case of void nothing gets returned, so ? doesn't solve anything.
The correct function signature would be
```ts
getSprite(name: string): Asset<SpriteData> | null
```

**Comment by lajbel** [comment]
It's true

**Comment by lajbel** [comment]
fixed [1f52c92](https://github.com/marklovers/kaplay/commit/1f52c9237a8cd54f4850561cc93960c3feed9aa8)
